ads/google/doubleclick.js

```diff
@@ -79,6 +79,15 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // If the ad layout is responsive, center the fixed size ad in the container.
+  if (global.context.layout == 'responsive') {
```

Comment:
Don't think so. Even in the multi-size case, I think we'll always want to center the ad response.